src/clojider_gatling_highcharts_reporter/reporter.clj

```diff
@@ -5,6 +5,12 @@
 (defn- flatten-one-level [coll]
   (mapcat #(if (sequential? %) % [%]) coll))
 
+(defn- failure-message [ex]
+  (if (instance? clojure.lang.ExceptionInfo ex)
+    ;; Don't report the data for a Clojure exception, as that would be too much for Gatling
+    (str "clojure.lang.ExceptionInfo: " (ex-message ex))
+    (str ex)))
```

Comment:
'Real' Gatling has the ability to report status code errors (`expected(200).received(403)`, or whatever) without an exception class, but in may cases the class is useful. Potentially we could define our own exception class in the highcharts reporter specifically for status code errors, and drop the class if that's what we get back. But thus far I've just been returning `(Exception. "useful error message here")`.

Reply:
Reporting the class for a base `Exception` / `ExceptionInfo` isn't useful in the way that, say, a `java.net.ConnectException` might be, so maybe if the class is the base `Throwable/Exception/ExceptionInfo` we don't bother with the class in the failure message?

Reply:
I ended up implementing this, so we only report the class if it's not one of the root/standard types.